dynawo/sources/Simulation/DYNSimulation.cpp

```diff
@@ -884,6 +886,7 @@ Simulation::updateParametersValues() {
 
 void
 Simulation::iterate() {
+  Timer timer("Simulation::iterate()");
```

Comment:
I have seen in ModelMulti that they are created inside #ifdef DEBUG conditions (so only in debug compilations). As they can be printed out only in this mode, I think it is quite a good idea. What do you think?
In this case, we could apply it here too.

Reply:
I totally agree.